presets/workspace/models/supported_models.yaml

```diff
@@ -6,34 +6,15 @@ models:
     tag: 0.0.1
 
   # Llama
-  - name: llama-2-7b
-    type: llama2-completion
-    runtime: llama-2
-    tag: 0.0.4
-  - name: llama-2-7b-chat
-    type: llama2-chat
-    runtime: llama-2
-    tag: 0.0.4
-  - name: llama-2-13b
-    type: llama2-completion
-    runtime: llama-2
-    tag: 0.0.4
-  - name: llama-2-13b-chat
-    type: llama2-chat
-    runtime: llama-2
-    tag: 0.0.4
-  - name: llama-2-70b
-    type: llama2-completion
-    runtime: llama-2
-    tag: 0.0.4
-  - name: llama-2-70b-chat
-    type: llama2-chat
-    runtime: llama-2
-    tag: 0.0.4
+  # TODO: Uncomment when ready to trigger llama3 build process
+  # - name: llama3.1-8b-instruct
+  #   type: text-generation
+  #   version: https://huggingface.co/meta-llama/Llama-3.1-8B-Instruct/commit/0e9e39f249a16976918f6564b8830bc894c89659
+  #   runtime: tfs
+  #   downloadAtRuntime: true
+  #   tag: 0.0.1
```

Comment:
nit: The `tag` field will not be used when `downloadAtRuntime` is enabled, as the kaito-base image will serve as the image for the inference workload.

Reply:
Yeah I realize, I'd like us to maintain tag for our own metadata sake. In next PR I'll add little check to our build pipeline to skip over building models with "downloadAtRuntime: true"